### ISSUE TITLE
BMesh Elements node is made an error

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#1774]
![prtscr capture_88](https://user-images.githubusercontent.com/3952687/30235738-64d5f9c6-954f-11e7-94bc-9e1db8c0e528.png)

I think this is a bug.

Error:
Node BMesh Elements had exception BMesh data of type BMesh has been removed

**Comment by enzyme69** [comment]
Changing v0, v1, v2 also giving error.

![prtscr capture_89](https://user-images.githubusercontent.com/3952687/30235796-8d1735ca-9550-11e7-94ff-bee5be50d993.png)

**Comment by enzyme69** [comment]
Was trying to make sense of masking when the data is BMesh type.
[bmesh_operation_001_2017_09_09_01_18.zip](https://github.com/nortikin/sverchok/files/1289448/bmesh_operation_001_2017_09_09_01_18.zip)

**Comment by zeffii** [comment]
the item socket in ListItem  is not a mask, it is an index.

![image](https://user-images.githubusercontent.com/619340/30238159-ca69b902-9541-11e7-877b-8b51e7455dfe.png)

**Comment by zeffii** [comment]
![image](https://user-images.githubusercontent.com/619340/30238197-8b112fd2-9542-11e7-9c07-5afff9c00c67.png)

The BMesh Elements node lets you pick a subset of the mesh topology.

**Comment by enzyme69** [comment]
​Thanks Dealga, much appreciated this explanation.

**Comment by enzyme69** [comment]
​However, still having issue with v0, v1, v2, and paremeters always easily
giving me red error on the node. Currently Github is down, so I cannot go
to Github to reply, this is using email.

**Comment by zeffii** [comment]
@enzyme69  quite right. (however the zip you linked to would never have given any sensible results either)

Now it gives this rather unhelpful error

```python
File "sverchok\nodes\scene\BMOperatorsMK2.py", line 87, in process
    exec(op)
  File "<string>", line 1, in <module>
Node BMesh Ops 2 had exception BMesh data of type BMesh has been removed
```

**Comment by zeffii** [comment]
@Kosvor2   it's your node, any ideas?

**Comment by zeffii** [comment]
these being alpha/beta nodes generally suggests that there's limited support for usage.

All the more reason to get cracking using scripted node to learn .  i really think that there's many many examples now..

**Comment by Kosvor2** [comment]
Sverchok nodetree must be updated as whole for this Bmesh nodes to work properly:

![bmesh_nodes123](https://user-images.githubusercontent.com/22656834/30240744-76022662-958f-11e7-9c21-072fa182f1bd.png)

**Comment by nortikin** [comment]
is it poke the same as alt+P shortcut?

**Comment by Kosvor2** [comment]
Yes, its a bmesh poke operator: https://docs.blender.org/api/2.79/bmesh.ops.html#bmesh.ops.poke

**Comment by nortikin** [comment]
so i made this in scripted node light too

**Comment by enzyme69** [comment]
I still think the BMesh Ops and its (not working, error prone) properties are a bit of sad. I cannot get v0, v1, etc to work, no matter how many times updating the node tree.

I looked at older thread and realized I have always tried to Poke with different height, etc. Didn't work. Error then.
https://github.com/nortikin/sverchok/issues/1611

**Comment by enzyme69** [comment]
@nortikin You have the SNL Poke? I am reading BMesh Ops... so Blender devs actually relies on this Ops to generate and model?

**Comment by nortikin** [comment]
@enzyme69 devs are relies on library and use functions for objects, yes.
you can use poke in SNL templates now

**Comment by nortikin** [comment]
next step vectorization of poke height

**Comment by enzyme69** [comment]
@nortikin Cannot see it inside SNL. I have latest SV.

**Comment by Kosvor2** [comment]
@enzyme69  Poke Offset works when Bmesh list is created from object, and indeed doesnt works when it created from Sverchok data: 

![poke_offset123](https://user-images.githubusercontent.com/22656834/30337513-ef4a9b52-9801-11e7-822b-a6f5e6ad0762.png)

Dont ask, idkw.

**Comment by nortikin** [comment]
https://github.com/nortikin/sverchok/blob/master/node_scripts/SNLite_templates/Alt_P_alike_divider.py
please, look at this

**Comment by enzyme69** [comment]
Thanks @Kosvor2 
![prtscr capture](https://user-images.githubusercontent.com/3952687/30355813-ef643de2-9878-11e7-88a2-d103baf674fd.png)

Thanks @nortikin 
Checking yours SNL Poke --- why name it so funky Alt P? What is that?

**Comment by zeffii** [comment]
jimmy, read!!    https://github.com/nortikin/sverchok/issues/1774#issuecomment-328368200
